profiles/common/modules/custom/nexteuropa_varnish/README.md

```diff
@@ -0,0 +1,111 @@
+About Varnish
+=============
+Varnish is a very fast reverse-proxy system which serves static 
+files and anonymous page views based on the previously processed
+requests.
+
+Nexteuropa Varnish
+==================
+The Nexteuropa Varnish module provides functionality which allows to
+send customized HTTP request to the Varnish server based on the
+configured 'purge rules'.
+The main purpose of those requests is to invalidate the Varnish cache to
+reflect recently published content changes.
+
+## Requirements
+This feature can be enabled only with the support of the QA/Maintenance
+team.
+The following environment specific variables have to be configured
+before enabling the feature:
+- 'nexteuropa_varnish_request_user' - a string with the username
+- 'nexteuropa_varnish_request_password' - a string with the password
+- 'nexteuropa_varnish_http_targets', - an array with the urls ex. 'http://localhost'
+- 'nexteuropa_varnish_tag', - a string with the tag
+- 'nexteuropa_varnish_request_method' - a string with the HTTP request method
+- 'nexteuropa_varnish_http_timeout' - a float representing the maximum number 
```

Comment:
The documentation specifies that the variable needs to be configured but then it is not checked in the hook requirements.